src/framework/uicomponents/view/popupview.cpp

```diff
@@ -71,6 +71,9 @@ PopupView::PopupView(QQuickItem* parent)
 
 PopupView::~PopupView()
 {
+    if (m_window) {
+        m_window->setOnHidden(std::function<void()>());
+    }

```

Comment:
Occasionally the "onHidden" handler would crash if it activated after this object had been deleted